Formula/glib.rb

```diff
@@ -51,63 +59,20 @@ def install
     mkdir "build" do
       system "meson", *args, ".."
       system "ninja", "-v"
-      system "ninja", "install", "-v"
+      system "meson", "install", "--no-rebuild", "--skip-subprojects", "include,lib"
       rewrite_shebang detected_python_shebang, *bin.children
     end
 
-    # ensure giomoduledir contains prefix, as this pkgconfig variable will be
-    # used by glib-networking and glib-openssl to determine where to install
-    # their modules
-    inreplace lib/"pkgconfig/gio-2.0.pc",
-              "giomoduledir=#{HOMEBREW_PREFIX}/lib/gio/modules",
-              "giomoduledir=${libdir}/gio/modules"
-
-    if OS.mac?
-      # `pkg-config --libs glib-2.0` includes -lintl, and gettext itself does not
-      # have a pkgconfig file, so we add gettext lib and include paths here.
-      gettext = Formula["gettext"].opt_prefix
-      inreplace lib+"pkgconfig/glib-2.0.pc" do |s|
-        s.gsub! "Libs: -L${libdir} -lglib-2.0 -lintl",
-                "Libs: -L${libdir} -lglib-2.0 -L#{gettext}/lib -lintl"
-        s.gsub! "Cflags: -I${includedir}/glib-2.0 -I${libdir}/glib-2.0/include",
-                "Cflags: -I${includedir}/glib-2.0 -I${libdir}/glib-2.0/include -I#{gettext}/include"
-      end
-    end
-
-    # `pkg-config --print-requires-private gobject-2.0` includes libffi,
-    # but that package is keg-only so it needs to look for the pkgconfig file
-    # in libffi's opt path.
-    libffi = Formula["libffi"].opt_prefix
-    inreplace lib+"pkgconfig/gobject-2.0.pc" do |s|
-      s.gsub! "Requires.private: libffi",
-              "Requires.private: #{libffi}/lib/pkgconfig/libffi.pc"
-    end
-
     bash_completion.install Dir["gio/completion/*"]
   end
 
   def post_install
-    (HOMEBREW_PREFIX/"lib/gio/modules").mkpath
+    libglib = Formula["libglib"]
+    include.make_relative_symlink(libglib.opt_include)
+    lib.make_relative_symlink(libglib.opt_lib)
   end
```

Comment:
Make symlinks to let the downstream formula access `Formula["glib"].opt_lib` and `Formula["libglib"].opt_lib`.